Monika After Story/game/script-stories.rpy

```diff
@@ -1015,3 +1038,295 @@ label mas_scary_story_revenant:
+    m 2wud "The smile on his wife's face melted into anger as she revealed that she was the snow woman he spoke of."
+    m 4efc "She reprimanded him for breaking his promise and would have killed him if it weren't for the sake of their children."
+    m 4efd "She told the man that he better treat their children well or she would be back to deal with him."
+    m 4dsd "The next instant she vanished, never to be seen again."
+    if renpy.random.randint(1,3) == 1 or persistent._mas_pm_likes_spoops or mas_full_scares:
+        show monika 6ckc
+        pause 0.2
+        show monika 4dsd
+    call mas_scary_story_cleanup
+    return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_story_database,
+            eventlabel="mas_scary_story_many_loves",
+            category=[store.mas_stories.TYPE_SCARY],
+            prompt="Many Loves",
+            unlocked=False
+        ),
+        code="STY"
+    )
+
+label mas_scary_story_many_loves:
+    call mas_scary_story_setup
+    m 4esa "There once was a young woman who appeared in a village one day to find a husband."
+    m 4eua "She was very beautiful and quickly attracted many suitors."
+    m 2eua "Eventually settling down with a husky fisherman."
+    m 2esd "The two had a happy marriage, but in less than a year, the husband wasted away and died."
+    m "The people in the village felt sorry for the young woman and comforted her the best they could."
+    m 4esa "A few months later, the woman married a burly woodcutter."
+    m 4dsd "The two lived happily together for a time, but he too withered away and died."
+    m 4eud "Some of the villagers thought that it was odd both husbands had died in the same fashion, but no one said anything, and comforted the girl for her poor luck."
+    m 2esc "A time later, the woman was married again, this time to a stout mason and they too appeared to have a happy marriage, but within a year, the woman was again a widow."
+    m "This time the villagers talked amongst themselves and felt something suspicious was going on, so a group of villagers set out to find the nearest shaman."
+    m "Once they found the shaman and told him their story, the shaman indicated he knew what was going on."
+    m 3euc "He called his assistant over, a young, well-built fellow, whispered into his ear and sent him off to return with the villagers."
+    m "Telling them not to worry, his assistant would get to the bottom of it."
+    m 2esc "When they returned to the village, the assistant called on the widow and not long afterwards they were married."
+    m 2efc "On the night of their wedding, the assistant placed a knife under his pillow and feigned sleep."
+    m 2esd "A little after midnight, the man felt a presence over him and a prick on his neck."
+    m 2dfc "The man grabbed the knife and stuck it into the thing on top of him."
+    if renpy.random.randint(1,20) == 1 and persistent._mas_pm_likes_spoops or mas_full_scares:

```

Comment:
can you group these conditions with parens, it looks confusing as to what is supposed to be true to trigger this.